### ISSUE TITLE
TTLock.setLockTime : Attempt to invoke virtual method 'long java.lang.Long.longValue()' on a null object reference

### ISSUE THREAD

**Issue by SDS712** [ttlock/ttlock_flutter#7]
The method `TTLock.setLockTime` worked fine when it was used right after initing the lock. Though now (2 days after initing) following error has been thrown while trying to use it:-

```
E/flutter (13603): [ERROR:flutter/lib/ui/ui_dart_state.cc(177)] Unhandled Exception: PlatformException(error, Attempt to invoke virtual method 'long java.lang.Long.longValue()' on a null object reference, null, java.lang.NullPointerException: Attempt to invoke virtual method 'long java.lang.Long.longValue()' on a null object reference
```

Trace:

```
E/flutter (13603): 	at com.ttlock.ttlock_flutter.TtlockFlutterPlugin.setLockTime(TtlockFlutterPlugin.java:606)
E/flutter (13603): 	at com.ttlock.ttlock_flutter.TtlockFlutterPlugin.doNextCommandAction(TtlockFlutterPlugin.java:354)
E/flutter (13603): 	at com.ttlock.ttlock_flutter.TtlockFlutterPlugin.doorLockCommand(TtlockFlutterPlugin.java:179)
E/flutter (13603): 	at com.ttlock.ttlock_flutter.TtlockFlutterPlugin.onMethodCall(TtlockFlutterPlugin.java:149)
E/flutter (13603): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:233)
E/flutter (13603): 	at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:85)
E/flutter (13603): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:692)
E/flutter (13603): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/flutter (13603): 	at android.os.MessageQueue.next(MessageQueue.java:323)
E/flutter (13603): 	at android.os.Looper.loop(Looper.java:141)
E/flutter (13603): 	at android.app.ActivityThread.main(ActivityThread.java:5603)
E/flutter (13603): 	at java.lang.reflect.Method.invoke(Native Method)
E/flutter (13603): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:774)
E/flutter (13603): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:652)
E/flutter (13603): )
E/flutter (13603): #0      StandardMethodCodec.decodeEnvelope   package:flutter/…/services/message_codecs.dart:582
E/flutter (13603): #1      MethodChannel._invokeMethod          package:flutter/…/services/platform_channel.dart:159
E/flutter (13603): <asynchronous suspension>
E/flutter (13603): #2      MethodChannel.invokeMethod           package:flutter/…/services/platform_channel.dart:332
E/flutter (13603): #3      TTLock.invoke                        package:ttlock_flutter/ttlock.dart:602
E/flutter (13603): #4      TTLock.setLockTime                   package:ttlock_flutter/ttlock.dart:423

```

**Comment by ttlock** [comment]
static void setLockTime(int timestamp, String lockData,
      TTSuccessCallback callback, TTFailedCallback failedCallback) {
    Map map = Map();
    map[TTResponse.timestamp] = timestamp;
    map[TTResponse.lockData] = lockData;
    invoke(COMMAND_SET_LOCK_TIME, map, callback, fail: failedCallback);
  }
  I d'not think you have set the value of TTResponse.timestamp. So the sdk get a null value of timestamp.

**Comment by SDS712** [comment]
Fixed as of `ttlock_flutter: ^0.0.6`
